pilot/pkg/model/push_context.go

```diff
@@ -768,6 +771,42 @@ func (ps *PushContext) GatewayServices(proxy *Proxy) []*Service {
 	return gwSvcs
 }
 
+type ExtensionProvider interface {
```

Comment:
dead code now. Agree it will not work after looking closer

Reply:
fixed